config/configure.in

```diff
@@ -563,6 +563,40 @@ else
   AC_MSG_RESULT([found $NUMS_INCLUDE])
 fi
 
+CURL=yes
+{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking ocaml curl library" >&5
+printf %s "checking ocaml curl library... " >&6; }
+CURL_INCLUDE=$(ocamlfind query -i-format curl 2> /dev/null)
+if [ $? -ne 0 ]; then
+  CURL=no
+fi
+CURL_LIB=$(ocamlfind query -a-format -predicates native curl 2> /dev/null)
+if [ $? -ne 0 ]; then
+  CURL=no
+fi
+CURL_CLIB=$(ocamlfind query -l-format -predicates native curl 2> /dev/null)
+if [ $? -ne 0 ]; then
+  CURL=no
+fi
+if test "$CURL" = "no"; then
+  AC_MSG_RESULT([no])
```

Comment:
Curl is now mandatory, so there should also be an `AC_MSG_ERROR([ocurl is missing])` here.